cookbooks/lib/languages/python_spec.rb

```diff
@@ -39,10 +39,9 @@
   end
 
   {
-    'python2.7' => '2.7.17',
-    'python3.6' => '3.6.10',
-    'python3.7' => '3.7.6',
-    'python3.8' => '3.8.1'
+    'python2.7' => '2.7.15',
+    'python3.6' => '3.6.7',
+    'python3.7' => '3.7.1'
```

Comment:
What about other distros, i propose to add some conditions here if xenial, if bionic etc

Reply:
Done! ✅ 
Thanks for the suggestions!